utils/utils.go

```diff
@@ -491,6 +499,82 @@ func GetAccountBalances(
 	return accountBalances, nil
 }
 
+// -------------------------------------------------------------------------------
+// ----------------- Helper struct for fetching account coins --------------------
+// -------------------------------------------------------------------------------
+
+// AccountCoinsRequest defines the required information to get an account's coins.
+type AccountCoinsRequest struct {
+	Account        *types.AccountIdentifier
+	Network        *types.NetworkIdentifier
+	Currencies     []*types.Currency
+	IncludeMempool bool
+}
+
+// AccountCoins defines an account's coins info in the tip.
+type AccountCoinsResponse struct {
+	Coins []*types.Coin
+}
+
+// GetAccountCoins calls /account/coins endpoint and returns an array of coins in the tip.
+func GetAccountCoins(
```

Comment:
Who will call this function?

Reply:
Oh I see, your later PR in rosetta-cli uses this.

Reply:
Yes https://github.com/coinbase/rosetta-cli/pull/290/files